code/online_challenges/src/hackerrank/jumping_on_the_clouds/jumping_on_the_clouds.c

```diff
@@ -0,0 +1,27 @@
+#include <stdio.h>
+
+int main()
+{
+    int n, i;
+    scanf ("%d", &n);
+    int c[n];
+    for (i=0;i<n;++i)
+    {
+        scanf ("%d", &c[i]);
+    }
+    int jump = 0;
+    for (i=0;i<n;++i)
```

Comment:
@anshuman1510 Did you reformat the file? The spacing doesn't look good.

Reply:
Sir, it has 4 space format could you please tell how to reformat the file so that spacing looks good.